README.md

```diff
@@ -128,7 +128,13 @@ Inertia.configure do |config|
 end
 ~~~
 
-## Configuring Axios to work with Rails CSRF protection
+## Javacript setup
+
+### Turbolinks
+
+IMPORTANT!! InertiaJS and Turbolinks do not play nicely together since they both intercept AJAX requests. If you want to use InertiaJS, you should leave Turbolinks out of your application.
```

Comment:
In my case the problem was, that we still has parts that use Turbolinks and we need it inside.
What we end up doing is:

```ruby
redirect_to happy_path, turbolinks: false
```

Reply:
i like that better than "don't use this with turbolinks at all"... i'm added this in #5 , i'd like us to confirm it and then we'll add it to the README... thanks for letting us know!